troposphere/template_generator.py

```diff
@@ -307,10 +308,11 @@ def _get_function_type(self, function_name):
 
     def _import_all_troposphere_modules(self):
         """ Imports all troposphere modules and returns them """
+        dirname = os.path.join(os.path.dirname(__file__))
         module_names = [
             pkg_name
             for importer, pkg_name, is_pkg in
-            pkgutil.walk_packages(__file__)
+            pkgutil.walk_packages([dirname], prefix="troposphere.")
```

Comment:
@cwgreene I looked at this earlier today and did have a question I was going to pose after I had time looking at the walk_packages interface. But I'll just ask it. I thought the ```prefix="troposphere"``` was added to limit the walk. So I would have thought keeping it and removing the ```pkg_name.startswith``` was the change you were going to make and the main change in this PR. But you removed the prefix and kept the startswith. Just wanted to verify that is what you intended to do.